x/stakingmiddleware/module.go

```diff
@@ -0,0 +1,161 @@
+package stakingmiddleware
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+
+	"github.com/notional-labs/composable/v6/x/mint/simulation"
```

Comment:
Shouldn't be here.

Reply:
If you are not using simulation in this module (it only appears to be using mint module's sim methods) you can remove the AppModuleSimulation interface assertion and remove the SimApp methods.

Reply:
thanks. removed 	"github.com/notional-labs/composable/v6/x/mint/simulation"